spec/spec_acceptance_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'beaker-rspec'
```

Comment:
Whoops. I think this accounts for some of the confusion earlier -- I was editing this file and not `spec/spec_helper_acceptance.rb`.